src/gerbil/runtime/util.ss

```diff
@@ -294,6 +294,9 @@ namespace: #f
                 rest))
       iv)))
 
+(def (drop l k)
+  (if (zero? k) l (drop (cdr l) (- k 1))))
```

Comment:
fxzero, fx1- they shouldn't be anything else.

Reply:
also, this will fault for negative. 

here is a better implementation:
```
(def (drop l k)
   (if (fxpositive? k) (drop (cdr l) (fx1- k)) l))
```